<?xml version="1.0" ?>
<launch>    
  
  <!-- map and localization system -->
  <arg name="no_nav" default="false"/>  
  <arg name="map_file" default="casa"/>
  <group unless="$(arg no_nav)">    
    <node pkg="map_server" type="map_server" name="map_server" args="$(find marrtino_r3d)/maps/$(arg map_file).yaml" output="screen"/> 
    <node pkg="amcl" type="amcl" name="amcl" output="screen">
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/amcl_config.yaml"/>
    </node>     
  </group>    
   <!--
 
  <node pkg="rodney_missions" type="rodney_missions_node.py" name="rodney_missions" output="screen">
    <rosparam command="load" file="$(find rodney_missions)/config/config.yaml"/>
  </node>
  
  <node pkg="rodney" type="rodney_node" name="rodney" args="-m $(find rodney)/maps/$(arg map_file)_patrol.yaml" output="screen">
    <rosparam command="load" file="$(find rodney)/config/config.yaml"/> 
  </node>
 
  
  -->
  
 
  
  
  <!-- Navigation --> 
  <group unless="$(arg no_nav)">   
    <node pkg="move_base" type="move_base" name="move_base" respawn="false" output="screen">
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/base_local_planner_params.yaml"/>
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/costmap_common_params.yaml" ns="global_costmap"/>
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/costmap_common_params.yaml" ns="local_costmap"/>
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/global_costmap_params.yaml"/>
      <rosparam command="load" file="$(find marrtino_r3d)/config/rodney/local_costmap_params.yaml"/>       
     <!-- <remap from="cmd_vel" to="demand_vel"/>  -->
    </node>   
  </group>
   
</launch>
